xdsl/ir/core.py

```diff
@@ -1668,6 +1668,14 @@ def erase_block(self, block: int | Block, safe_erase: bool = True) -> None:
         block = self.detach_block(block)
         block.erase(safe_erase=safe_erase)
 
+    def clone(self) -> Region:
+        """
+        clone the entire region into a new one.
```

Comment:
```suggestion
        Clone the entire region into a new one.
```

Docstring comments start with a capital letter.